openpype/hosts/resolve/utility_scripts/openpype_startup.scriptlib

```diff
@@ -18,4 +18,4 @@ if openpype_startup_script ~= nil then
     else
         print("Launch script not found at: " .. script)
     end
-end
```

Comment:
I presume this was committed by mistake.

Reply:
This script has some conflict of missing the line with my branch after merging with the develop

Reply:
but this should not be part of this PR. Please reverse the change back to the state before the change was made.